### ISSUE TITLE
Editor rejects two equally named methods even when they have different quantity of arguments

### ISSUE THREAD

**Issue by npasserini** [uqbar-project/wollok#289]
![image](https://cloud.githubusercontent.com/assets/4633913/10403303/0fab49fa-6ea1-11e5-858d-af9a4729ee0e.png)

Shouldn't this be allowed?

**Comment by javierfernandes** [comment]
```
def duplicatedMethod(WMethodDeclaration m) {
		// can we allow methods with same name but different arg size ? 
		if (m.declaringContext.members.filter(WMethodDeclaration).exists[it != m && it.name == m.name])
			m.report(WollokDslValidator_DUPLICATED_METHOD, DUPLICATED_METHOD)
	}
```

Anyway. I'll check for arity. It's easy

**Comment by javierfernandes** [comment]
Done in "dev-quickfixes" which will be merged to "dev" for 1.2 release
